index.html

```diff
@@ -6,19 +6,20 @@
 
   <link rel="stylesheet" href="styles/reset.css" />
   <link rel="stylesheet" href="styles/style.css" />
+  <link href='https://fonts.googleapis.com/css?family=Ubuntu' rel='stylesheet' type='text/css'>
```

Comment:
@jrob8577 yup, super simple. google fonts add link rel and then call the font in your css. the thing to worry about is load times.

ta-da: https://www.google.com/fonts